frida_tools/tracer.py

```diff
@@ -581,6 +605,10 @@ def _create_trace_script(self):
 var pending = [];
 var timer = null;
 
+state.newline = String.fromCharCode(10);
```

Comment:
```suggestion
```

Can we escape this instead of requiring use of a constant?

Reply:
It is crucial not to write any string containing a slash and an 'n'.  Calling String.fromCharCode(10) enables us to avoid such strings.  What did you have in mind?

Reply:
I was thinking we could escape the user's `\n` instead of requiring them to use a constant.